### PR TITLE
docs: installation process improve

### LINKED ISSUE #798: Multiple package managers guide
Hi team! 
I hope for usage of several popular package managers besides npm.

### PR DESCRIPTION
**Pull request by alexey-pd**
Closes #798

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,22 +54,55 @@ Effector implements business logic with ease for Javascript apps **(React/React
...
+
+with **yarn**
+```sh
+yarn add effector effector-vue
+```
+
+with **npm**
 ```sh
-npm add effector effector-vue
+npm install effector effector-vue
```
[the remark this answers is not in the document]
Comment (alexey-pd) [reply README.md:105]: install mostly used - i can change it to add if needed

[the remark this answers is not in the document]
Comment (sergeysova) [reply README.md:105]: I think it is better to use specialized command

[the remark this answers is not in the document]
Comment (alexey-pd) [reply README.md]: moved to npm add

**Comment by alexey-pd** [comment]
is any changes needed here?

**Pull request closed without merging**